Sources/active-win/main.swift

```diff
@@ -5,19 +5,19 @@ func toJson<T>(_ data: T) throws -> String {
 	return String(data: json, encoding: .utf8)!
 }
 
+// Show accessibility permissions screen if not enabled; required to get the complete window title
 if !AXIsProcessTrustedWithOptions(["AXTrustedCheckOptionPrompt": true] as CFDictionary) {
-	print("BActive-Win requires permission for accessibility in System Preferences > Security & Privacy > Privacy > Accessibility.")
-	exit(0)
+	print("Active-Win requires permission for accessibility in System Preferences > Security & Privacy > Privacy > Accessibility.")
+	exit(1)
 }
 
 let frontmostAppPID = NSWorkspace.shared.frontmostApplication!.processIdentifier
 let windows = CGWindowListCopyWindowInfo([.optionOnScreenOnly, .excludeDesktopElements], kCGNullWindowID) as! [[String: Any]]
 
-let windowImage = CGWindowListCreateImage(.null, .optionIncludingWindow, windows[0][kCGWindowNumber as String] as! CGWindowID, [.boundsIgnoreFraming, .bestResolution])
-
-if windowImage == nil {
-	print("Active-Win requires permission for screen recording in System Preferences > Security & Privacy > Privacy > Screen Recording.")
-	exit(0)
+// Show screen recording permissions screen if not enabled; required to get the complete window title
+if CGWindowListCreateImage(.null, .optionIncludingWindow, windows[0][kCGWindowNumber as String] as! CGWindowID, [.boundsIgnoreFraming, .bestResolution]) == nil {
```

Comment:
> windows[0][kCGWindowNumber as String] as! CGWindowID

What if there are no windows?

Reply:
I hope this is the proper way to check if first array element exists in swift ;)

Reply:
I hope this is the proper way to check for the first array element in swift.